cookbooks/travis_internal_base/spec/ssh_spec.rb

```diff
@@ -62,13 +68,14 @@
   strictmodes yes
   syslogfacility AUTH
   tcpkeepalive yes
-  usedns yes
+  usedns no
   uselogin no
-  usepam 1
+  usepam yes
   useprivilegeseparation yes
-  versionaddendum
+  fingerprinthash SHA256
+  versionaddendum none
   x11displayoffset 10
-  x11forwarding no
+  x11forwarding yes
```

Comment:
From `streamlocalbindmask` through `x11forwarding`, I'm unsure how much should be asserted.  If the values that aren't (arguably) about security differ between trusty and xenial, I don't think the spec should fail.